tests/Unit/Domain/Test_NormalDotFlux.cpp

```diff
@@ -0,0 +1,346 @@
+// Distributed under the MIT License.
```

Comment:
You can see the correct diff for this file when stepping through the commits one-by-one